specification/datacollaboration/resource-manager/Microsoft.DataCollaboration/preview/2020-05-04-preview/DataCollaboration.json

```diff
@@ -0,0 +1,5736 @@
+        },
+        "name": {
+          "description": "Operation name for display purposes",
+          "type": "string"
+        },
+        "origin": {
+          "description": "origin of the operation",
+          "type": "string"
+        },
+        "properties": {
+          "$ref": "#/definitions/OperationMetaPropertyInfo",
+          "description": "properties for the operation meta info",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "OperationModelProperties": {
+      "description": "Properties on operations",
+      "type": "object",
+      "properties": {
+        "description": {
+          "description": "Description of the operation for display purposes",
+          "type": "string"
+        },
+        "operation": {
+          "description": "Name of the operation for display purposes",
+          "type": "string"
+        },
+        "provider": {
+          "description": "Name of the provider for display purposes",
+          "type": "string"
+        },
+        "resource": {
+          "description": "Name of the resource type for display purposes",
+          "type": "string"
+        }
+      }
+    },
+    "OperationResponse": {
+      "description": "Response for long running operation",
+      "required": [
+        "status"
+      ],
+      "type": "object",
+      "properties": {
+        "endTime": {
+          "format": "date-time",
+          "description": "start time",
+          "type": "string"
+        },
+        "error": {
+          "$ref": "#/definitions/DataShareErrorInfo",
+          "description": "The error property when status is failed."
+        },
+        "startTime": {
+          "format": "date-time",
+          "description": "start time",
+          "type": "string"
+        },
+        "status": {
+          "description": "Operation state of the long running operation.",
+          "enum": [
+            "Accepted",
+            "InProgress",
+            "TransientFailure",
+            "Succeeded",
+            "Failed",
+            "Canceled"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "status",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "Participant": {
+      "description": "Readonly participant information",
+      "type": "object",
+      "properties": {
+        "participantId": {
+          "description": "Unique id for identifying a participant",
+          "type": "string",
+          "readOnly": true
+        },
+        "proposalApprovalStatus": {
+          "description": "Current approval status",
+          "enum": [
+            "Pending",
+            "Approved",
+            "Active",
+            "Revoked"
+          ],
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "proposalApprovalStatus",
+            "modelAsString": true
+          }
+        },
+        "userEmail": {
```

Comment:
have you been through a privacy review regarding the exposure of emails and names in the service?

Reply:
Not yet, but we based many of the models on ADS which are GA and undergo regular privacy reviews so I don't expect ours review to be different. Any reason it shoudl be?